pkg/config/config.go

```diff
@@ -679,6 +685,23 @@ func load(config Config, origin string, loadSecret bool) error {
 		}
 	}
 
+	// If this variable is set, we'll use DefaultPython for the Python version,
+	// ignoring the python_version configuration value.
+	if ForceDefaultPython != "" {
```

Comment:
That works for now, but someone might get surprised if they compile the Agent with `-X REPO_PATH/pkg/config.ForceDefaultPython=false` and that still forces the Python version.
Do you think it's worth explicitly parsing the option as a boolean?

Reply:
Worth the change